src/components/BusinessOnboarding/Step1.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import CompanyCard from './CompanyCard';
+
+import {
+    Background,
+    MainContainer
+} from './BusinessOnboarding.styles';
+
+
+// Theres gotta be a better way..
```

Comment:
You can just write this text directly into the jsx instead of keeping some object holding all the text... like <CompanyCard input="Enter a company name. eg. Google" ....... />